src/kubernetes_api_objects/role.rs

```diff
@@ -52,6 +52,17 @@ impl Role {
         self.inner.metadata = metadata.into_kube();
     }
 
+    #[verifier(external_body)]
+    pub fn take_policy_rules(&mut self, other_role: Role)
```

Comment:
I think this one does not have to be external_body. You should be able to implement it with a get and a set.

Reply:
It is a minor thing. We can deal with this later.